src/components/Content/Content.js

```diff
@@ -9,19 +9,69 @@ import {
   Modal,
   Typography,
 } from '@material-ui/core'
+import {
+  withStyles,
+} from '@material-ui/core/styles';
 import GitHubIcon from '@material-ui/icons/GitHub'
 import TwitterIcon from '@material-ui/icons/Twitter'
 import LinkedInIcon from '@material-ui/icons/LinkedIn'
+import TextField from '@material-ui/core/TextField'
 import Link from '@material-ui/core/Link'
 import useStyles from './Content.styles'
 import { AwesomeButtonSocial } from 'react-awesome-button'
 import 'react-awesome-button/dist/styles.css'
 import Profile from '../Profile'
 
+const SearchTextField = withStyles((theme) => {
+  return {
+    root: {
+      '& .MuiOutlinedInput-root': {
+        '& fieldset': {
+          borderColor: theme.palette.secondary.dark,
+        },
+        '&:hover fieldset': {
+          borderColor: theme.palette.primary.contrastText,
+        },
+        '&.Mui-focused fieldset': {
+          borderColor: theme.palette.primary.contrastText,
+        },
+      },
+    },
+  }
+})(TextField);
+
 function Content(props) {
   const { edges } = props.data.allContributorsJson
   const classes = useStyles()
   const [modal, setModal] = React.useState(false)
+  const [filteredParticipants, setFilteredParticipants] = React.useState(edges)
+  const [searchText, setSearchText] = React.useState('')
+
+  const handleSearch = ({ target }) => {
+    setSearchText(target.value);
+  }
+
+  const getGithubUsernameFromURL = (URL) => {
+    return URL.substring(19).toLowerCase()
```

Comment:
```suggestion
    return URL.split('.com/')[1]
```
Use this instead. Because we are already using this functionality while calling an API. If you still think what you did was a better approach then, please validate it with an explanation. Thanks

Reply:
If this being used already, then we can go with this approach itself.

Reply:
Cool!